src/Modules/Frontend/templates/base/Head.html.twig

```diff
@@ -3,8 +3,8 @@
 <head>
   <meta charset="utf-8" />
   <meta name="generator" content="Fork CMS" />
-{#  {{ meta|raw }}#}
-{#  {{ metaCustom|raw }}#}
+{# {{ meta|raw }} #}
+{# {{ metaCustom|raw }} #}
```

Comment:
@carakas why is this commented?

Reply:
Probably something I forgot while working on the frontend

Reply:
Is fixed in https://github.com/forkcms/forkcms/pull/3586